mux/state.go

```diff
@@ -193,7 +194,7 @@ func (b *StateBroadcaster) run(ttl time.Duration) {
 				for output := range b.outputs {
 					close(output)
 				}
-				ticker.Stop() //to avoid goroutine leak
+				break
```

Comment:
This break will break the select, but not the for loop
https://golang.org/ref/spec#Break_statements

Is this what's expected? (I have thinking issues today)